examples/planning/python/simple_client.py

```diff
@@ -0,0 +1,270 @@
+import grpc
+import sys
+import yaml
+import os
+import os.path
+from typing import List
+
+ROOT_DIR = os.path.dirname(os.path.abspath(__file__))
+DATA_DIR = os.path.join(os.path.dirname(ROOT_DIR), "data")
+BUILDFILES_DIR = os.path.join(ROOT_DIR, "build")
+if BUILDFILES_DIR not in sys.path:
+    sys.path.append(BUILDFILES_DIR)
+
+from build.types_pb2 import (
+    Constraints,
+    Model,
+    SystemConf,
+    Conf,
+    PositionConstraint,
+    AngleBetweenVectorsConstraint,
+    PlanContextId,
+)
+from build.generate_id_pb2 import (
+    RegisterPlanContextRequest,
+    RegisterPlanContextResponse,
+)
+from build.generate_id_pb2_grpc import PlanContextRegistryStub
+from build.builder_pb2 import StartBuildFromConfsRequest, StartBuildResponse
+from build.builder_pb2_grpc import IrisBuilderStub
+
+
+def make_position_constraint_msg(
+    frame_A: str,
+    frame_B: str,
+    p_AQ_lower: List[float],
+    p_AQ_upper: List[float],
+    position_BQ: List[float],
+) -> PositionConstraint:
+    """Make a Protobuf message for a position constraint on a point Q atached
+    to a frame B, measured and expressed in frame A.
+
+    Args:
+        frame_A (str): The frame in which point Q's position is measured
+        frame_B (str): The frame to which point Q is rigidly attached.
```

Comment:
Can you clarify? If the frame doesn't exist in the model, it will throw, and the response will contain some information about the exception. If a user inputs the "wrong" frame, i.e., not the one they had hoped to constrain but still otherwise valid, there's not much we can do to infer their original intent.

Reply:
> it will throw, and the response will contain some information about the exception
ok -- that's fine. I would put that into the docstring.